Rubrik/Private/Submit-Request.ps1

```diff
@@ -1,4 +1,4 @@
-﻿function Submit-Request($uri,$header,$method,$body)
+﻿function Submit-Request($uri,$header,$method = $($resources.Method) ,$body)
```

Comment:
Curious why this change was required?

Reply:
Due to the change made to line 15 for the Invoke-WebRequest. The original version was locked into whatever the $resources item was. I needed Submit-Request to not be dependent on the $resources collection (be more generic). However, to maintain the functionality I changed the method to default to $resources.Method .